### ISSUE TITLE
Absence of XHTML index should not be reported as a warning

### ISSUE THREAD

**Issue by elmimmo** [w3c/epubcheck#1266]
epubcheck reports a missing XHTML `title` as a warning. 

```
$ epubcheck --locale en -mode exp EPUB
Validating using EPUB version 3.2 rules.
WARNING(RSC-017): …/001.xhtml(4,8): Warning while parsing file: The "head" element should have a "title" child element.
```

The [HTML Standard says](https://html.spec.whatwg.org/dev/semantics.html#the-title-element) (bolds are mine):

> If it's reasonable for the Document to have no title, then **the title element is probably not required.**

[and](https://html.spec.whatwg.org/dev/semantics.html#the-head-element):

> The title element is a required child in most situations, **but when a higher-level protocol provides title information,** e.g. in the Subject line of an email when HTML is used as an email authoring format, **the title element can be omitted.**

which seems like a provision for EPUB precisely. I would only find it natural that most EPUBs’ XHTML lacked a title element, never parsed nor displayed, serving no purpose.

Therefore, this specific warning in the validation output is noise, IMHO.

**Comment by rdeltour** [comment]
We discussed this in #448. The conclusion was that there is no _higher-level protocol_ providing a **document**’s title in EPUB, only the general title of the publication. So we report this as a `WARNING`.

I'm closing this issue as a duplicate, but feel free to keep on commenting, here or maybe preferably in #448!

**Comment by elmimmo** [comment]
Well, there is a way to have epubcheck shut up about it just for me? I cannot find documentation on `--customMessages`.

**Comment by rdeltour** [comment]
Unfortunately the `customMessages` option won't help much. This check is implemented as a parsing warning (RSC-017), so you can't disable this one without disabling all the checks reported as `RSC-017` (which would be quite discouraged).

The only way currently would be to comment out the line in the source code, and build a custom version of EPUBCheck.
